hot-desk/src/main/java/com/exadel/sandbox/booking/repository/BookingRepository.java

```diff
@@ -29,5 +28,6 @@
     @Query(value = "select b from Booking b join b.seat s join s.floor f join f.office o where o.id = :officeId")
     List<Booking> findBookingsByOfficeId(Long officeId);
 
-    List<Booking> findBookingsByEmployeeId(Long employeeId);
+    List<Booking> getBookingsByEmployeeId(Long employee_id);
```

Comment:
We can't use snake_case in java code we should use camelCase

Reply:
oo sorry It was by chance). Done!